apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/util/ShardUtils.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c)  2018-2021. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.util;
+
+import com.apollocurrency.aplwallet.apl.core.dao.TransactionalDataSource;
+import com.apollocurrency.aplwallet.apl.core.service.appdata.DatabaseManager;
+import com.apollocurrency.aplwallet.apl.core.shard.ShardManagement;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author andrew.zinchenko@gmail.com
+ */
+@NoArgsConstructor(access = AccessLevel.NONE)
+public class ShardUtils {
```

Comment:
Set `@Vetoed ` annotation here to prevent weld occasional discovery

Reply:
Ok